Sources/StreamChatUI/CommonViews/CreateChannelButton/CreateChannelButton.swift

```diff
@@ -6,7 +6,7 @@ import StreamChat
 import UIKit
 
 /// A Button subclass that should be used for creating new channels.
-open class ChatChannelCreateNewButton: _Button, AppearanceProvider {
+open class CreateChannelButton: _Button, AppearanceProvider {
```

Comment:
Same here? Is it a button for creating a new "channel" or creating a new `ChatChannel`? 🤔 Probably the former?

Reply:
In this case, it is clear that it should be `CreateChatChannelButton` right? `ChatChannelCreateButton` does not make sense while reading it

Reply:
Done ✅